datasets/afvalwijzer/dataset.json

```
@@ -0,0 +1,227 @@
+{
+    "id": "afvalwijzer",
+    "type": "dataset",
+    "authorizationGrantor": "n.v.t.",
+    "theme": [
+        "Duurzaamheid en milieu, wonen"
+    ],
+    "homepage": "https://data.amsterdam.nl",
+    "owner": "Gemeente Amsterdam, Afval en Grondstoffen",
+    "dateModified": "",
+    "spatialDescription": "Amsterdam",
+    "version": "1.0.0",
+    "title": "Afvalwijzer",
+    "language": "nl",
+    "dateCreated": "2022-01-01",

```

This should work. I'll see if I can fix this on our side.

---

Nope, we should really use a date+time here for compatibility with DCAT.

```suggestion
    "dateCreated": "2022-01-01T00:00:00",
```

---

unfortunately still not accepting `"dateCreated": "2022-01-01T00:00:00+01:00",`

---

Works locally, let me try again:

```suggestion
    "dateCreated": "2022-01-01T00:00:00+01:00",
```